deployment/helm/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.1.0
 description: React to events from Falco
 name: falco-talon
-version: 0.1.0
+version: 0.2.0
```

```suggestion
version: 0.1.0
```

The project is still in alpha stage, this tag is there to prepare the first release, we can keep it.

---

done.